R/default_parameters.R

```diff
@@ -0,0 +1,11 @@
+#' Default parameters for SEM.
+#'
+#' Default parameters for SEM.
+#'
+#' @format A list with lots of entries:
+#' \describe{
+#'   \item{price}{price, in US dollars}
+#'   \item{carat}{weight of the diamond, in carats}
+#'   ...
+#' }
+"default_parameters"
```

Comment:
Ah I thought that this sort of documentation went in `R/data.R` Oh but it looks like I made a mistake there, so let's delete that one or something.